File: Lua/FizzBuzz.lua
for x = 1, 100 do
    if (x % 3 == 0 and x % 5 == 0) then
        print("FizzBuzz")
    elseif (x % 3 == 0) then
        print("Fizz");
    elseif (x % 5 == 0) then
        print("Buzz");
    else
        print(x or fb)
    end
end

File: JavaScript/FizzBuzz.js
for (x = 1; x < 101; x++) {
  var fb = "";
  if (x % 3 == 0) fb = "Fizz";
  if (x % 5 == 0) fb += "Buzz";
  if (!fb) {
    console.log(x);
  } else console.log(fb);
}
